lib/components/screens/likes_retweets/retweets/cubit/retweets_cubit.dart

```diff
@@ -0,0 +1,116 @@
+import 'dart:convert';
+
+import 'package:bloc/bloc.dart';
+import 'package:built_collection/built_collection.dart';
+import 'package:dart_twitter_api/twitter_api.dart';
+import 'package:freezed_annotation/freezed_annotation.dart';
+import 'package:harpy/api/api.dart';
+import 'package:harpy/components/components.dart';
+import 'package:harpy/core/core.dart';
+
+part 'retweets_cubit.freezed.dart';
+
+class RetweetsCubit extends Cubit<PaginatedState<RetweetedUsersData>>
+    with RequestLock, HarpyLogger {
+  RetweetsCubit({required this.tweetId})
+      : super(const PaginatedState.loading());
+  late String tweetId;
+  UserSortBy sort = UserSortBy.fromJsonString(
+    app<UserListSortPreferences>().listSortOrder,
+  );
+
+  Future<void> _request({bool clearPrevious = false}) async {
+    if (clearPrevious) {
+      emit(const PaginatedState.loading());
+    }
+
+    final users = await app<TwitterApi>()
+        .tweetService
+        .retweets(id: tweetId, count: 100)
+        .handleError(twitterApiErrorHandler)
+        .then(
+          (tweets) => createUsers(tweets, sort, null),
+        );
+
+    if (users.isNotEmpty) {
+      final data = RetweetedUsersData(
+        tweetId: tweetId,
+        users: users.toBuiltList(),
+      );
+      emit(
+        PaginatedState.data(
+          data: data,
+        ),
+      );
+    }
+  }
+
+  Future<void> loadRetweetedByUsers() async {
+    emit(const PaginatedState.loading());
+
+    await _request();
+  }
+
+  // following the correct rule but ide takes something else is happening
+  // ignore: use_setters_to_change_properties
```

Comment:
The lint is basically telling you not to wrap a single setter into a method. Instead you can just use the setter directly

Reply:
Ah, that makes sense. I wasn't sure how to fix this issue at the time. Thanks!